### DISCUSSION TITLE
Fooocus update incoming

### DISCUSSION THREAD

**Discussion by ILikeToasters** [toutjavascript/Fooocus-Log-Viewer#25] [General]
Just wanted to give a heads up that there is a Fooocus update coming in the next day or two. I have not had a chance to test with the release candidate yet since I have been changing some things around and don't have the log viewer set back up yet. I do know there is some changes to the information in the log so that the complete prompts are now shown. Just wanted to give a heads up.

**Comment by ILikeToasters** [comment]
I went ahead and got mine back up and running to test it and it does still work fine but obviously missing the new full prompt that was added.

**Comment by toutjavascript** [comment]
Thanks for information and testing
I am trying to update Fooocus Log Viewer and FoooXus Fooocus Extender to comply with 2.4 release
